### PR TITLE
French Translation

### PR DESCRIPTION
**Pull request by Desoroxxx**
Thought a French translation could be nice, so I did one

### CODE REVIEW

File: src/main/resources/assets/rainglow/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "rainglow.config.title": "Configuration de Rainglow",
+  "rainglow.config.current_colours": "Couleurs actuellement activées :",
+  "rainglow.config.colours_to_apply": "Couleurs à appliquer :",
+  "rainglow.config.more": "plus",
```
Comment (ix0rai) [inline src/main/resources/assets/rainglow/lang/fr_fr.json:5]:
```suggestion
  "rainglow.config.more": "Plus",
```

**Review by ix0rai — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/rainglow/lang/fr_fr.json (hunk 1 above)
Comment (Desoroxxx) [reply src/main/resources/assets/rainglow/lang/fr_fr.json:5]: I tried being faithful to the original translation, which did not have capitalization there but it guess it was a mistake, could I get confirmation it was a mistake?
https://github.com/ix0rai/rainglow/blob/master/src/main/resources/assets/rainglow/lang/en_us.json#L5

Comment (ix0rai) [reply src/main/resources/assets/rainglow/lang/fr_fr.json:5]: yes, my mistake
I'll fix that today

Comment (Desoroxxx) [reply src/main/resources/assets/rainglow/lang/fr_fr.json]: > yes, my mistake I'll fix that today

Okay, merged the suggestion

**Pull request merged**